superduperdb/components/vector_index.py

```diff
@@ -54,6 +61,26 @@ def on_load(self, db: Datalayer) -> None:
             self.compatible_listener = t.cast(
                 Listener, db.load('listener', self.compatible_listener)
             )
+        if CFG.cluster.vector_search.type == "pg_vector":
```

Comment:
This logic should be isolated in the PgVector class (vector_searcher).

Reply:
@blythed you want me to move this logic to `PosgresVectorSearcher` class, and then call it from here.

Reply:
> This logic should be isolated in the PgVector class (vector_searcher).

@blythed you want me to move this logic to `PosgresVectorSearcher` class, and then call it from here. right?

Reply:
@blythed working on it